### PR TITLE
Update to Minecraft 1.11.2

### PR DESCRIPTION
**Pull request by tyra314**
Updates your awesome mod to 1.11.2

May contain bugs. I have only tested it for a bit in the dev environment.

### CODE REVIEW

**Comment by tyra314** [comment]
Whuups, I just saw, asie did the same already 😢

**Comment by tyra314** [comment]
I noticed, asies PR lacks combine recipe. In my version it works. So some merge would be nice, right?

**Comment by Hunternif** [comment]
Hey tyra314, thank you for your work!
I'll make sure to take a look at both pull request when I have time.

**Comment by osipxd** [comment]
I've tried to merge your pull requerst with mine but Atlas Combine recipe still not works.

**Comment by osipxd** [comment]
Issue #107 created.

**Comment by tyra314** [comment]
You missed one commit.

https://github.com/tyra314/AntiqueAtlas/commit/d0dfc6562b5ba42bfed80e2e8b3e845435b334d0

From what I saw in debugging this, the returned ItemStack will be used as output in the crafting recipe, but as it is a reference to one of the input ItemStacks in the crafting grid, it will get deleted once the ItemStacks in the grafting grid will be deleted. So after crafting, there is a sync between server and client, which will result in the output Atlas being deleted.

TL;DR The important bit is, to make a copy of the returned firstAtlas ItemStack in getCraftingResult().

**Comment by osipxd** [comment]
Thanks, I'll add it

**Comment by tyra314** [comment]
Side note: Issue #92 still persist, I haven't been able to figure it out 😢
